2024/day02/solution.md

```
@@ -0,0 +1,201 @@
+
+- ```bash
+    mkdir directoryName
+    ``` 
+- ### Make a directory in a specific location.
+
+#### Sample output image: 
+![mkdir devops:](./images/Screenshot%202024-10-10%20180434.png)
+
+---
+
+## Examples:
```

Markdownlint

Multiple headings with the same content

(MD024, no-duplicate-heading)